### PR TITLE
API refactor LeftAndMain_Menu.ss into individually overridable components

### PR DESCRIPTION
**Pull request by tractorcow**
Replaces https://github.com/silverstripe/silverstripe-framework/pull/4795

### CODE REVIEW

File: admin/templates/Includes/LeftAndMain_MenuStatus.ss [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+	<a href="$LogoutURL" class="logout-link font-icon-logout" title="<%t LeftAndMain_Menu_ss.LOGOUT 'Log out' %>"></a>
+	<% with $CurrentMember %>
+		<span>
+			<%t LeftAndMain_Menu_ss.Hello 'Hi' %>
+			<a href="{$AbsoluteBaseURL}admin/myprofile" class="profile-link">
+				<% if $FirstName && $Surname %>$FirstName $Surname<% else_if $FirstName %>$FirstName<% else %>$Email<% end_if %>
+			</a>
+		</span>
+	<% end_with %>
+</div>
```
Comment (dhensby) [inline admin/templates/Includes/LeftAndMain_MenuStatus.ss]: no new line? Oh @tractorcow how could you

**Comment by jonom** [comment]
Thanks for moving this to 3. Per previous comment would you mind splitting the `cms-logo` and `cms-login-status` divs out in to separate includes instead of housing them together? Would give people flexibility to override just one or the other.

**Comment by tractorcow** [comment]
Good idea. I'm out of action for a while due to sickness... Jono want to
make a new PR?

**Comment by jonom** [comment]
Hope you feel better soon! https://github.com/silverstripe/silverstripe-framework/pull/5185

**Pull request closed without merging**